kernel/dev/vga/graphics.hpp

```diff
@@ -50,4 +53,17 @@ void pixel(uint32_t x, uint32_t y, uint32_t color);
  */
 void putrect(uint32_t x, uint32_t y, uint32_t w, uint32_t h, uint32_t color);
 
+/**
+ * @brief Fill the backbuffer with '0'.
+ *
+ */
+void clearbuffer();
```

Comment:
I would prefer something more verbose / clear like `resetDoubleBuffer` since `clearbuffer` could imply clearing the normal framebuffer.